meson.build

```diff
@@ -20,13 +20,18 @@ program_git = find_program('git', required: true)
 
 cc = meson.get_compiler('c')
 
+host_machine_cpu_family = host_machine.cpu_family()
+if host_machine_cpu_family == 'x86'
+    host_machine_cpu_family = 'i386'
```

Comment:
It'd be nice if 'x86' can cover both 'i386' and 'x86_64' (preferred) depending on the system (word-size).

Reply:
If the system is 64-bit, `host_machine.cpu_family()` will return `x86_64`. If the system is 32-bit, it will return `x86`. Thus, I don't think it is necessary. Also, `host_machine` doesn't seem to have word size in it.

The standard `host_machine.cpu_family()` values for Meson are listed in this [page](https://mesonbuild.com/Reference-tables.html#cpu-families).

Reply:
I was thinking about user-given arch names like for cross-compilation.

Reply:
The `cpu_family` should only use standard values. If the user wants to be more specific, he will define `cpu` field in the cross compilation file:

```INI
[host_machine]
system = 'linux'
cpu_family = 'x86'
cpu = 'i586'
endian = 'little'
```

Since we use `cpu_family`, it shouldn't matter. See this [page](https://mesonbuild.com/Cross-compilation.html#machine-entries).